source/features/address/ui/AddressSummary.tsx

```diff
@@ -70,7 +76,9 @@ const AddressSummary = (props: AddressSummaryProps) => {
                 <div className={styles.infoLabel}>
                   {translate('address.totalWithdrawn')}
                 </div>
-                <div className={styles.infoValue}>{props.totalWithdrawn} ADA</div>
+                <div className={styles.infoValue}>
+                  {props.totalWithdrawn} ADA
+                </div>
```

Comment:
Btw. why are half of the changes only formatting related? This is very confusing for reviews :(

Reply:
I am always using 'yarn prettier:format' before my commits/push . @DominikGuzei. I assume most of the old changes weren't formatted previously.